Tools/scripts/summarize_stats.py

```diff
@@ -62,7 +62,7 @@ def get_defines(filepath: Path, prefix: str = "SPEC_FAIL"):
                     continue
                 line = line[len(start) :]
                 name, val = line.split()
-                defines[int(val.strip())].append(name.strip())
+                defines[int(val.strip().strip("()"))].append(name.strip())
```

Comment:
```suggestion
                defines[int(val.strip().removesuffix("()"))].append(name.strip())
```

This is probably safer, unless there is a need to remove things like `(())` that I'm missing.

Reply:
The defines for the flags look like `#define HAS_PASSTHROUGH_FLAG (4096)` - this line is to strip the parens around the number. We could do `resmovesuffix(")")` and `removeprefix("(")`?

Could also do the whole thing with a regex if that's preferable, but it would be a little unweildy.

Reply:
I see.  I misunderstood what this needed to do.  I think this is fine.